presto-main/src/main/java/com/facebook/presto/sql/gen/PageFunctionCompiler.java

```diff
@@ -119,7 +119,7 @@ public PageFunctionCompiler(Metadata metadata, int expressionCacheSize)
             projectionCache = CacheBuilder.newBuilder()
                     .recordStats()
                     .maximumSize(expressionCacheSize)
-                    .build(CacheLoader.from(cacheKey -> compileProjectionInternal(cacheKey.sqlFunctionProperties, cacheKey.rowExpression, Optional.empty())));
```

Comment:
So there is no concern that projections will generate large amounts of compiled code? Not 100% sure what the projection cache contains so I am lazily asking you.

Reply:
Still wondering about this as well